subsys/bluetooth/mesh/Kconfig

```diff
@@ -288,13 +291,17 @@ config BT_MESH_CRPL
 
 config BT_MESH_MSG_CACHE_SIZE
 	int "Network message cache size"
-	default 10
+	default 32
 	range 2 65535
 	help
-	  Number of messages that are cached for the network. This helps
-	  prevent unnecessary decryption operations and unnecessary
-	  relays. This option is similar to the replay protection list,
-	  but has a different purpose.
+	  Number of messages that are cached by the node to avoid acting on the
+	  recently seen duplicate messages. This option is similar to
+	  the replay protection list, but has a different purpose. Network message
+	  cache helps prevent unnecessary decryption operations. This also prevents
+	  unnecessary relaying and helps in getting rid of relay loops. Setting
+	  this value to a very low number can result in increased unnecessary
+	  network traffic. Setting this value to a large number can impact the
+	  processing time for each received network PDU and increase RAM footprint.
```

Comment:
```suggestion
	  this value to a very low number can cause unnecessary
	  network traffic. Increasing this value impacts the
	  processing time for each received network PDU and increase RAM footprint.
```

Reply:
I will improve this a bit, compared to your suggestion.